### PR TITLE
[DOCS] Update "using the compiler".

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/using-the-compiler.rst [hunk 1]
```diff
@@ -10,13 +10,17 @@ Using the Commandline Compiler
...
 Using ``solc --help`` provides you with an explanation of all options. The compiler can produce various outputs, ranging from simple binaries and assembly over an abstract syntax tree (parse tree) to estimations of gas usage.
 If you only want to compile a single file, you run it as ``solc --bin sourceFile.sol`` and it will print the binary. If you want to get some of the more advanced output variants of ``solc``, it is probably better to tell it to output everything to separate files using ``solc -o outputDirectory --bin --ast --asm sourceFile.sol``.
 
-Before you deploy your contract, activate the optimizer while compiling using ``solc --optimize --bin sourceFile.sol``. By default, the optimizer will optimize the contract for 200 runs. If you want to optimize for initial contract deployment and get the smallest output, set it to ``--runs=1``. If you expect many transactions and don't care for higher deployment cost and output size, set ``--runs`` to a high number.
+Before you deploy your contract, activate the optimizer while compiling using ``solc --optimize --bin sourceFile.sol``.
```
Comment (axic) [inline docs/using-the-compiler.rst]: while -> when?

File: docs/using-the-compiler.rst [hunk 2]
```diff
@@ -113,7 +127,7 @@ Input Description
 .. code-block:: none
 
     {
-      // Required: Source code language, such as "Solidity", "serpent", "lll", "assembly", etc.
+      // Required: Source code language, such as "Solidity", "vyper", "lll", "assembly", etc.
```
Comment (axic) [inline docs/using-the-compiler.rst]: Perhaps `vyper` -> `Vyper`?

**Review by axic — DISMISSED** [review]
Fine to merge apart form those typos/remarks.

File: docs/using-the-compiler.rst (hunk 1 above)
Comment (ChrisChinchilla) [reply docs/using-the-compiler.rst]: Both kinda work fine.

**Pull request merged**